### ISSUE TITLE
Connect to Mongo with TLS

### ISSUE THREAD

**Issue by sagikazarmark** [errbit/errbit#1263]
TLS is the next buzzword of the last few years and will be for the years coming.

Some services (like MongoDB on compose.io) come with TLS enabled by default (unfortunately sometimes with self-signed certs).

It would be really great if Errbit could:

- connect to MongoDB using TLS
- optionally disable verification
- accept a CA for self-signed certs

After a little bit of reading I think it should be possible, but my initial attempts failed, so I could really use some help to achieve the above.

Official support would even be better. :wink:

Thanks in advance!

**Comment by rud** [comment]
Good idea. What did you try, how did it fail? Do you have code/test to reproduce the problem? 

One thing to certainly check is whether it is a supported version of MongoDB you are connecting to, as the most recent releases are not yet supported. Just as a heads up

**Comment by sagikazarmark** [comment]
Well, to be honest it was not the most professional way what I tried: I was hacking the configuration locally in a docker image (and it looks like it is lost now thanks to a recent Docker reset).

But basically I was trying to override some configuration in [this](https://github.com/errbit/errbit/blob/master/config/mongo.rb) file according to [this](https://docs.mongodb.com/mongoid/5.2/tutorials/mongoid-installation/#configuration) documentation.

Although the version in Gemfile is set to 5.0.2, I was hoping that the `ssl_` config options were not added after that.

I also checked the available Mongo versions on Compose.io: 3.2.10, 3.2.11, 3.2.18, 3.4.10
(I tried 3.2.18)
